default/python/turn_events/turn_events.py

```diff
@@ -31,4 +31,35 @@ def execute_turn_events():
         if fo.create_field(field_type, x, y, size) == fo.invalid_object():
             print >> sys.stderr, "Turn events: couldn't create new field"
 
+    gsd = fo.get_galaxy_setup_data()
+    # creating monsters
```

Comment:
Sorry for the nitpick, but this comment should better go above the 'gsd = fo.get_galaxy_setup_data()` line (it's the caption for the entire section after all) :wink: